geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaInputFormat.scala

```diff
@@ -151,7 +167,23 @@ class GeoMesaInputFormat extends InputFormat[Text, SimpleFeature] with LazyLoggi
   var table: GeoMesaTable = null
 
   private def init(conf: Configuration) = if (sft == null) {
-    val params = GeoMesaConfigurator.getDataStoreInParams(conf)
+    val dsParams = GeoMesaConfigurator.getDataStoreInParams(conf)
```

Comment:
I don't think you need to make any changes to this method - the mock parameters should get passed through and the data store should create the mock connection for us. It might be b/c you are hard-coding the instance name - I think we can avoid that above though.